_articles/scripting/Typescript/typescript-modifier.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Modifiers in Typescript
+author: Shush
+steamId: 76561197994333648
+date: 07.03.2021
+---
+
+Modifiers are an extremely important part of almost any Dota addons. They allow you to modify certain properties of your hero, deal damage to it over time, or apply various effects on it. Like abilities, we'll also create them in Typescript.
+
+We'll use an easy example which should cover a lot of common concepts for modifiers. The example would be Skywrath's Ancient Seal, which, when cast on an enemy, simply applies the modifier to it. The modifier applies the Silenced state on the enemy, and reduces its magic resist property by a percentage.
```

Comment:
'The example is', just use present tense

Reply:
'The example would be Skywrath's Ancient Seal, which, when cast on an enemy, simply applies the modifier to it.' Unclear: the examples is an ability? I thought this was a modifier tutorial? Suggestion: 'This example is Skywrath's Ancient seal, which is an ability that simply applies a modifier to an enemy.'

Reply:
Great suggestion, replaced it.